client/application.cpp

```diff
@@ -408,36 +409,81 @@ void application::initialize()
 
 	// TODO get it from application info
 	xr_actionset = xr::actionset(xr_instance, "actions", "Actions");
-	std::vector<XrActionSuggestedBinding> bindings;
 
-	for (const auto & [name, type]: oculus_touch)
-	{
-		std::string name_without_slashes = std::string(name).substr(1);
+	// TODO better way to handle bindings for different vendors/flavors of vendor
+	char property_chararr[PROP_VALUE_MAX + 1];
+	__system_property_get("ro.product.manufacturer", property_chararr);
```

Comment:
This, and the associated include, should be guarded by ifdefs. We try to support a PC version of the client, and this breaks compilation.

Reply:
I added ifdefs

Reply:
also made some fallback variable as well, just a placeholder for future